src/pt/spectralscan/src/eu/kanade/tachiyomi/extension/pt/spectralscan/NexusScan.kt

```diff
@@ -0,0 +1,311 @@
+            .addQueryParameter("genre", "")
+            .addQueryParameter("type", "")
+            .addQueryParameter("view", "grid")
+            .build()
+        return GET(url, ajaxHeaders())
+    }
+
+    override fun popularMangaParse(response: Response): MangasPage {
+        return parseMangaListResponse(response)
+    }
+
+    // ==================== Latest ==========================
+
+    override fun latestUpdatesRequest(page: Int): Request {
+        val url = "$baseUrl/ajax/load-mangas/".toHttpUrl().newBuilder()
+            .addQueryParameter("page", page.toString())
+            .addQueryParameter("sort", "latest")
+            .addQueryParameter("q", "")
+            .addQueryParameter("genre", "")
+            .addQueryParameter("type", "")
+            .addQueryParameter("view", "grid")
+            .build()
+        return GET(url, ajaxHeaders())
+    }
+
+    override fun latestUpdatesParse(response: Response): MangasPage {
+        return parseMangaListResponse(response)
+    }
+
+    // ==================== Search ==========================
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        var sortValue = "popular"
+        var genreValue = ""
+        var typeValue = ""
+
+        filters.forEach { filter ->
+            when (filter) {
+                is SelectFilter -> {
+                    val value = filter.selected()
+                    if (value.isNotEmpty()) {
+                        when (filter.parameter) {
+                            "sort" -> sortValue = value
+                            "genre" -> genreValue = value
+                            "type" -> typeValue = value
+                        }
+                    }
+                }
+                else -> {}
+            }
+        }
+
+        val url = "$baseUrl/ajax/load-mangas/".toHttpUrl().newBuilder()
+            .addQueryParameter("page", page.toString())
+            .addQueryParameter("sort", sortValue)
+            .addQueryParameter("q", query)
+            .addQueryParameter("genre", genreValue)
+            .addQueryParameter("type", typeValue)
+            .addQueryParameter("view", "grid")
+            .build()
+
+        return GET(url, ajaxHeaders())
+    }
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        return parseMangaListResponse(response)
+    }
+
+    // ==================== Details =======================
+
+    override fun mangaDetailsParse(response: Response): SManga {
+        val document = response.asJsoup()
+
+        return SManga.create().apply {
+            title = document.selectFirst("h1.item-title, h1")?.text()?.trim() ?: ""
+            thumbnail_url = document.selectFirst("img.item-cover-image")?.absUrl("src")
+            description = document.select(".item-description p, .item-description").text()
+
+            genre = document.select(".genre-tag, .hero-genres .genre-tag").joinToString { it.text().trim() }
```

Comment:
`.text()` is already trimmed.
```suggestion
            genre = document.select(".genre-tag, .hero-genres .genre-tag").joinToString { it.text() }
```